explorer/client/src/__tests__/unit.test.ts

```diff
@@ -23,4 +26,21 @@ describe('Unit Tests', () => {
             expect(timeAgo(1735693989100, timeNow)).toEqual('< 1 sec');
         });
     });
+
+    describe('presentBN', () => {
+        it('handles increasing numbers', () => {
```

Comment:
You can also use `it.each` here which will spin out all of the right tests for you and keep your test case clean: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Good idea; I've implemented this.